src/content/en/2021/page-weight.md

```diff
@@ -152,7 +152,7 @@ The request distribution shows that the difference between desktop and mobile is
 
 The difference between current results for this year and last actually shows a tiny decrease in the average number of GET requests across most of the percentiles. Let's hope that trend continues downward.
 
-Something else worth noting: the median request on desktop at this time is the same [as last year](../2020/page-weight#page-requests) (74), yet the page weight has ticked up (141 kb).
+Something else worth noting: the median request on desktop at this time is the same [as 2 years ago](../2019/page-weight#page-requests) (74), yet the page weight has ticked up (141 kb).
```

Comment:
Not sure why you changed this? It was the same as 2020, and while it may also have been the same in 2019, I think it's fine to compare to "last year".

```suggestion
Something else worth noting: the median request on desktop at this time is the same [as last year](../2020/page-weight#page-requests) (74), yet the page weight has ticked up (141 kb).
```

Does that mean the Japanese translation needs to change too?

Reply:
@tunetheweb 
I didn't find 2020/page-weight#page-requests.
Does it exist?
what line in en/2020/page-weight?

Reply:
I changed it back to `../2020/page-weight#requests` (without the `page-`)

Reply:
@tunetheweb Thanks, I fixed it.